composer.json

```diff
@@ -33,7 +33,7 @@
 		}
 	],
 	"require": {
-		"php": ">5.3.0,~5.5.0"
+		"php": ">5.3.0,~5.6.0"
```

Comment:
Ubuntu 12.04 LTS (Precise Pangolin) has PHP 5.3.10 and has support untill 2017.
Ubuntu 14.04 LTS (Trusty Tahr) has PHP 5.5.12 and has support until 2019.
See: https://launchpad.net/ubuntu/+source/php5

I would suggest to stick our Ubuntu reference installation with 5.3 to 5.5 for the moment.